lib/eclipse/EclipseState/Schedule/TimeMap.cpp

```diff
@@ -189,57 +239,71 @@ namespace Opm {
         }
     }
 
-    void TimeMap::addFromDATESKeyword( const DeckKeyword& DATESKeyword ) {
+    std::time_t TimeMap::timeTFromEclipse(const DeckRecord &dateRecord) {
+        static const std::string errorMsg("The datarecord must consist of the four values "
```

Comment:
Do you here suggest that the time component from the data record, here `timeItem`, be discarded in the calculation of `date`?